### ISSUE TITLE
Add a SetEnvVars API method

### ISSUE THREAD

**Issue by stephenh** [toddbluhm/env-cmd#361]
Hi! Currently I'm using `env-cmd` to load `.env` for script commands in `package.json`, but when tests are invoked from vscode/webstorm, I'm using Jest's `globalSetup` with this file:

```
import { GetEnvVars } from "env-cmd";

export default async function globalSetup() {
  Object.entries(await GetEnvVars()).forEach(([key, value]) => (process.env[key] = value));
}
```

I've copy/pasted this incantation into a few projects now, and I'd like to be able to do just:

```
import { SetEnvVars } from "env-cmd";

export default SetEnvVars;
```

Does that seem reasonable? Happy to work on a PR if so. Thanks!

**Comment by toddbluhm** [comment]
Yeah, more than happy to accept a PR related to this. I don't really know many people using the API, so any API requests or ideas are welcomed!
